metrics/comet/README.md

```diff
@@ -0,0 +1,127 @@
+# Metric Card for COMET
+
+## Metric description
+
+Crosslingual Optimized Metric for Evaluation of Translation (COMET) is an open-source framework used to train Machine Translation metrics that achieve high levels of correlation with different types of human judgments. 
+
+
+## How to use 
+
+COMET takes 3 lists of strings as input: `sources` (a list of source sentences), `predictions` (a list of candidate translations) and `references` (a list of reference translations). 
+
+```python
+from datasets import load_metric
+comet_metric = load_metric('comet')
+source = ["Dem Feuer konnte Einhalt geboten werden", "Schulen und Kindergärten wurden eröffnet."]
+hypothesis = ["The fire could be stopped", "Schools and kindergartens were open"]
+reference = ["They were able to control the fire.", "Schools and kindergartens opened"]
+comet_score = comet_metric.compute(predictions=hypothesis, references=reference, sources=source)
+```
+It also has several optional arguments:
+
+`cuda`: a boolean -- if set to `True`, COMET will run using GPU; else, it will run using CPU. The default value is `True`.
+
+`show_progress`a boolean -- if set to `True`, progress updates will be printed out. The default value is `False`.
+
+`model`: the COMET model to be used. Will default to `wmt-large-da-estimator-1719`. Alternate models that can be chosen include `wmt20-comet-da`, `wmt20-comet-qe-da`, `wmt21-comet-mqm`, `wmt21-cometinho-da`, `wmt21-comet-qe-mqm` and `emnlp20-comet-rank`. 
+
+More information about model characteristics can be found on the [COMET website](https://unbabel.github.io/COMET/html/models.html).
+
+## Output values
+
+The COMET metric outputs two lists:
+
+`samples`: a list of dictionaries with `src`, `mt`, `ref` and `score`
```

Comment:
The latest version of `comet` doesn't return the samples. In https://github.com/huggingface/datasets/pull/3945, I had to remove it when updating the metric. Given that the metric has an extremely low usage (only 20 downloads per week) I think this is fine.

Feel free to merge this PR and we can discuss this and update the metric card if needed in https://github.com/huggingface/datasets/pull/3945